deepsea.spec.in

```diff
@@ -745,6 +745,8 @@ the README for more information.
 
 %files qa
 %{_libexecdir}/deepsea/qa
+%{_libexecdir}/deepsea/qa/osd-config
+%{_libexecdir}/deepsea/qa/osd-config/ovh
```

Comment:
drop these two lines - they're not needed (`%{_libexecdir}/deepsea/qa` already brings in everything under `qa/`)